modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/EditAPIApplicationSchema.testcase

```diff
@@ -0,0 +1,216 @@
+@component-name = "portal-headless"
+definition {
+
+	property custom.properties = "feature.flag.LPS-178642=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Object";
+	property testray.main.component.name = "Headless Builder";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given New API Application created") {
```

Comment:
The rule in headless qa team is that we do not modify the GWT. The task that moved to setUp() can be the existing Given. Could you please help remove the `Given+ And Given` in the setUp()?

Reply:
Hi @yanan , thanks for letting me know, I was totally unaware of this rule, do you have any documentation about this so I can stay on top of it for future PR submissions? I simply looked at other Headless files in the same path "`[UI] DefineAPISchemasWithinAnAPIGroup`" and thought it was expected, and did the same since all tests would need these initial steps. Thank you for reporting this, it is indeed news.